src/test/java/tennisclub/dao/CourtDaoTest.java

```diff
@@ -19,13 +24,128 @@
 
 	@Test
 	@Transactional
```

Comment:
If all of the methods are @Transactional, isn't it just better to put the annotation before the definition of the class? 👀

Reply:
True!